ingresspipe/schemas/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .explorer import *
```

Comment:
`import *` is generally advised against in Python best practices. Even though its more verbose, it's good to explicitly list out which objects are being imported — e.g.:

```python
from .explorer import SchemaExplorer, SchemaValidator
from some_other_module import SomeClass, some_function
```

Reply:
Thanks @jaeddy! I've made changes accordingly in [this commit](https://github.com/Sage-Bionetworks/HTAN-data-pipeline/pull/230/commits/8d58b75344aefc991c4c142667bdaac05139c5c8).